source/elements/oneVPL/source/index.rst

```diff
@@ -3,313 +3,38 @@
+GPUs, and other accelerators. It provides device discovery and selection in
+media centric and video analytics workloads and API primitives for zero-copy
+buffer sharing. oneVPL is backwards and cross-architecture compatible to ensure
+optimal execution on current and next generation hardware without source code
+changes.
 
+---------------------
 oneVPL API versioning
-=====================
+---------------------
 
-As a successor of MSDKt oneVPL API version starts from 2.0.
+oneVPL is the successor to |msdk_full_name|. oneVPL API versioning starts from
+2.0. There is a correspondent version of |msdk_full_name| API which is used as a basis
+for oneVPL and defined as ``MFX_LEGACY_VERSION`` macro.
 
-Experimental API's in oneVPL are protected with the following macro::
+Experimental APIs in oneVPL are protected with the following macro::
 
    #if (MFX_VERSION >= MFX_VERSION_NEXT)
 
 To use the API, define the ``MFX_VERSION_USE_LATEST`` macro.
 
-Acronyms and Abbreviations
-==========================
-.. include:: MSDK_acronyms.inc.rst
-
-Architecture
-============
-.. include:: MSDK_architecture.inc.rst
-
-Video Decoding
---------------
-.. include:: MSDK_decoding.inc.rst
-
-Video Encoding
---------------
-.. include:: MSDK_encoding.inc.rst
-
-Video Processing
-----------------
-.. include:: MSDK_processing.inc.rst
-
-Programming Guide
-=================
-.. include:: MSDK_prg.inc.rst
-
-Summary Tables
-==============
-.. include:: VPL_summary.inc.rst
-
-Appendicies
-===========
-.. include:: VPL_apnds.inc.rst
-
-
-oneVPL API Reference
-====================
-
-Basic Types
------------
-.. doxygentypedef:: mfxU8
-   :project: oneVPL
-
-.. doxygentypedef:: mfxI8
-   :project: oneVPL
-
-.. doxygentypedef:: mfxU16
-   :project: oneVPL
-
-.. doxygentypedef:: mfxI16
-   :project: oneVPL
-
-.. doxygentypedef:: mfxU32
-   :project: oneVPL
-
-.. doxygentypedef:: mfxI32
-   :project: oneVPL
-
-.. doxygentypedef:: mfxUL32
-   :project: oneVPL
-
-.. doxygentypedef:: mfxL32
-   :project: oneVPL
-
-.. doxygentypedef:: mfxU64
-   :project: oneVPL
-
-.. doxygentypedef:: mfxI64
-   :project: oneVPL
-
-.. doxygentypedef:: mfxF32
-   :project: oneVPL
-
-.. doxygentypedef:: mfxF64
-   :project: oneVPL
-
-.. doxygentypedef:: mfxHDL
-   :project: oneVPL
-
-.. doxygentypedef:: mfxMemId
-   :project: oneVPL
-
-.. doxygentypedef:: mfxThreadTask
-   :project: oneVPL
-
-.. doxygentypedef:: mfxChar
-   :project: oneVPL
-
-Typedefs
---------
-
-.. doxygentypedef:: mfxSession
-   :project: oneVPL 
-
-.. doxygentypedef:: mfxSyncPoint
-   :project: oneVPL
-
-.. doxygentypedef:: mfxLoader
-   :project: oneVPL 
-
-.. doxygentypedef:: mfxConfig
-   :project: oneVPL
-
-oneVPL Dispatcher API
----------------------
-
-.. include:: VPL_disp_api.inc.rst
-
-Enums
------
-
-.. include:: VPL_enums.inc.rst
-
-Structs
--------
-
-.. include:: VPL_structs.inc.rst
-
-Functions
----------------
-
-.. include:: VPL_functions.inc.rst
+.. toctree::
```

Comment:
Here is the full spec rendered as HTML: http://ansatnuc02.an.intel.com/rscohn1-spec/oneapi-spec/build/html/elements/oneVPL/source/index.html

Look at VPL table of contents in the left pane. You can only the versions section.  You cannot navigate to archtecture, or any of the other sections. Looking at your index.html, you have set the TOC as hidden, and have nested it inside the versioning section.

I suggest you move the versioning section into a separate file, add it to the TOC, and then make the TOC unhidden.

Reply:
Kristal will do the change in a separate commit. I will merge as-is.